src/plugins/conditionals/README.md

```diff
@@ -113,9 +113,9 @@ EOF
 echo "key = false" > `kdb file /examples/conditionals/sub`
 
 # should fail and yield an error
-kdb export /examples/conditionals ini
-#> sub/key = false
+kdb export /examples/conditionals simpleini
```

Comment:
oh, didn't know that.
i changed it to simpleini because it's easier to read in tutorial after ini started to support metakeys (the metakey would be printed to stdout too)
i could add a `nometa` flag to ini though.